pkg/executor/infoschema_reader.go

```diff
@@ -1792,7 +1783,7 @@ func keyColumnUsageInTable(schema model.CIStr, table *model.TableInfo, extractor
 			continue
 		}
 
-		if extractor != nil && extractor.Filter("constraint_name", filterIdxName) {
+		if !ex.HasConstraint(filterIdxName) {
```

Comment:
This makes no difference.
People forge to write the `extractor.Filter(` ... they can also forget to write `HasXXX`?

Reply:
Just trying to avoid using string literal here